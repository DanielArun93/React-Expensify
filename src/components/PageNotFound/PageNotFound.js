import React from 'react';
import ReactDOM from 'react-dom';
import { Link, NavLink } from 'react-router-dom';
class PageNotFound extends React.Component {
    render() {
        return (
            <div>
                <h1>Page not found 404!- <Link to="/">Go home</Link></h1>
            </div>
        )
    }
}

export default PageNotFound;