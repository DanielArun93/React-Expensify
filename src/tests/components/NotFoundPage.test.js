import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../components/PageNotFound/PageNotFound';

test('should render PageNotFound', () => {
    const wrapper = shallow(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
})