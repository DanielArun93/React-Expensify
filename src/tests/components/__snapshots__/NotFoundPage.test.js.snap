// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound 1`] = `
<div>
  <h1>
    Page not found 404!- 
    <Link
      replace={false}
      to="/"
    >
      Go home
    </Link>
  </h1>
</div>
`;
